src/engines/aurora/gui.h

```diff
@@ -115,10 +115,14 @@ class GUI : boost::noncopyable {
 	/** Callback that's triggered when a widget was activated. */
 	virtual void callbackActive(Widget &widget);
 
+	void addChild(GUI*);
+	void removeChild(GUI*);
 private:
 	typedef std::list<Widget *> WidgetList;
 	typedef std::map<Common::UString, Widget *> WidgetMap;
 
+	std::list<GUI*> _childrenguis;
```

Comment:
Hmm, that should probably be rather named "_childGUIs".

child, because GUIs is already the plural (and you generally only plural the last part in a compund word). And "GUIs", because acronyms/initialisms in the xoreos codebase are always uppercased when it's the 2nd+ part in a camelCase word (see also, for example "showFPS").